Gemfile

```diff
@@ -27,7 +27,7 @@ gem 'link_header'
 gem 'mail'
 gem 'mailjet'
 gem 'mas-assets', git: 'git@github.com:moneyadviceservice/mas-assets'
-gem 'meta-tags', github: 'moneyadviceservice/meta-tags'
+gem 'meta-tags', github: 'kpumuk/meta-tags', branch: 'master'
```

Comment:
Cool, but still good to add a brief note so anyone who looks at it in the meantime understands why we're tracking master.